compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -925,6 +925,7 @@ void quantize_const_inputs(luci::CircleNode *node, loco::DataType output_type)
     case luci::CircleOpcode::LOCAL_RESPONSE_NORMALIZATION:
     case luci::CircleOpcode::MEAN:
     case luci::CircleOpcode::PAD:
+    case luci::CircleOpcode::MIRROR_PAD:
     case luci::CircleOpcode::REDUCE_ANY:
```

Comment:
This is NOT alphabetical order. Please be aware and post another PR to FIX THIS!

Reply:
Sorry, I've noticed this too late. Here's PR: #6870